### PR TITLE
Add section on how to get help

### PR DESCRIPTION
**Pull request by qlik-aoh**
### Status
```
[ ] Under development
[X] Waiting for code review
[ ] Waiting for merge
```
### Information
```
[ ] Contains breaking changes
[ ] Contains new API(s)
[X] Contains documentation
[ ] Contains test
[ ] Contains examples
```

### CODE REVIEW

**Review by LinneaStovring-Nielsen — APPROVED** [review]

**Review by Bjornwa — APPROVED** [review]
Good - Much needed!

**Review by kjartan-qlik — APPROVED** [review]
Looks good, should community maybe be mentioned as well?

**Comment by qlik-aoh** [comment]
> Looks good, should community maybe be mentioned as well?

You're right, I'll add something about it.

**Comment by qlik-aoh** [comment]
@kjartan-qlik @Bjornwa @LinneaStovring-Nielsen 

Rewritten, referring to both Branch and Community.

File: README.md [hunk 1]
```diff
@@ -52,4 +52,11 @@ __Examples Disclaimer:__ The examples provided are just examples, therefore __us
 Please follow the instructions in [CONTRIBUTING.md](.github/CONTRIBUTING.md).
 
 ## Getting help
-For quickest assistance, questions is best asked in #sse on [Qlik Branch slack](https://qlik-branch.slack.com). For access, go to [Qlik Branch](https://developer.qlik.com/#squad) and qlik on [SLACK](https://qlikbranch-slack-invite.herokuapp.com/).
+1. Lok at the documentation and examples in this repo again.
```
Comment (LinneaStovring-Nielsen) [inline README.md]: Look

Comment (qlik-aoh) [reply README.md]: Thanks!

**Comment by qlik-aoh** [comment]
@DonThomasO Do you have time for to review?

**Pull request merged**